CODEOWNERS

```diff
@@ -25,3 +25,337 @@ trainer.py @muellerzr @SunMarc
+/src/transformers/models/table-transformer/*_modeling* @amyeroberts @qubvel
+/src/transformers/models/textnet/*_modeling* @amyeroberts @qubvel
+/src/transformers/models/timm_wrapper/*_modeling* @amyeroberts @qubvel
+/src/transformers/models/upernet/*_modeling* @amyeroberts @qubvel
+/src/transformers/models/van/*_modeling* @amyeroberts @qubvel
+/src/transformers/models/vit/*_modeling* @amyeroberts @qubvel
+/src/transformers/models/vit_hybrid/*_modeling* @amyeroberts @qubvel
+/src/transformers/models/vitdet/*_modeling* @amyeroberts @qubvel
+/src/transformers/models/vit_mae/*_modeling* @amyeroberts @qubvel
+/src/transformers/models/vitmatte/*_modeling* @amyeroberts @qubvel
+/src/transformers/models/vit_msn/*_modeling* @amyeroberts @qubvel
+/src/transformers/models/vitpose/*_modeling* @amyeroberts @qubvel
+/src/transformers/models/yolos/*_modeling* @amyeroberts @qubvel
+/src/transformers/models/zoedepth/*_modeling* @amyeroberts @qubvel
+
+# Audio models
+/src/transformers/models/audio-spectrogram-transformer/*_modeling* @eustlb
+/src/transformers/models/bark/*_modeling* @eustlb
+/src/transformers/models/clap/*_modeling* @eustlb
+/src/transformers/models/dac/*_modeling* @eustlb
+/src/transformers/models/encodec/*_modeling* @eustlb
+/src/transformers/models/hubert/*_modeling* @eustlb
+/src/transformers/models/mctct/*_modeling* @eustlb
+/src/transformers/models/mimi/*_modeling* @eustlb
+/src/transformers/models/mms/*_modeling* @eustlb
+/src/transformers/models/moshi/*_modeling* @eustlb
+/src/transformers/models/musicgen/*_modeling* @eustlb
+/src/transformers/models/musicgen_melody/*_modeling* @eustlb
+/src/transformers/models/pop2piano/*_modeling* @eustlb
+/src/transformers/models/seamless_m4t/*_modeling* @eustlb
+/src/transformers/models/seamless_m4t_v2/*_modeling* @eustlb
+/src/transformers/models/sew/*_modeling* @eustlb
+/src/transformers/models/sew-d/*_modeling* @eustlb
+/src/transformers/models/speech_to_text/*_modeling* @eustlb
+/src/transformers/models/speech_to_text_2/*_modeling* @eustlb
+/src/transformers/models/speecht5/*_modeling* @eustlb
+/src/transformers/models/unispeech/*_modeling* @eustlb
+/src/transformers/models/unispeech-sat/*_modeling* @eustlb
+/src/transformers/models/univnet/*_modeling* @eustlb
+/src/transformers/models/vits/*_modeling* @eustlb
+/src/transformers/models/wav2vec2/*_modeling* @eustlb
+/src/transformers/models/wav2vec2-bert/*_modeling* @eustlb
+/src/transformers/models/wav2vec2-conformer/*_modeling* @eustlb
+/src/transformers/models/wav2vec2_phoneme/*_modeling* @eustlb
+/src/transformers/models/wavlm/*_modeling* @eustlb
+/src/transformers/models/whisper/*_modeling* @eustlb
+/src/transformers/models/xls_r/*_modeling* @eustlb
+/src/transformers/models/xlsr_wav2vec2/*_modeling* @eustlb
+
+# Video models
+/src/transformers/models/timesformer/*_modeling* @Rocketknight1
+/src/transformers/models/videomae/*_modeling* @Rocketknight1
+/src/transformers/models/vivit/*_modeling* @Rocketknight1
+
+# Multimodal models
```

Comment:
I can take care of zero-shot object-detection models as well
owlvit, owlv2, grounding-dino, omdet-turbo

Reply:
Added!